### PR TITLE
Toasts on new connection/lost connection

### PR DESCRIPTION
**Pull request by lukasschreiber**
Closes https://github.com/reform-org/reform/issues/172

### CODE REVIEW

**Comment by mohe2015** [comment]
We should probably check the CI env and hide the toasts (except error toasts?) there as the disconnect toast breaks selenium interactions

**Comment by lukasschreiber** [comment]
Actually I think handling that only in the CI would be better, so that the tested code does not need to be modified for the CI to work - I have an idea on that

File: webapp/tests/selenium/main.ts [hunk 1]
```diff
@@ -11,6 +11,12 @@ export async function run() {
 	) {
 		let peer = await Peer.create(true);
 		await peer.driver.get("http://localhost:5173/");
+		peer.driver.executeScript(() => {
```
Comment (mohe2015) [inline webapp/tests/selenium/main.ts:14]: probably would need to be done at every page reload etc so I don't know if this is the most effective way. Also it's kinda unexpected that the tests suddenly change the page

Comment (lukasschreiber) [reply webapp/tests/selenium/main.ts:14]: Dann mach das mit der CI env...

**Pull request merged**